x-pack/plugins/cross_cluster_replication/public/app/services/api.js

```diff
@@ -74,6 +74,21 @@ export const createFollowerIndex = (followerIndex) => (
   httpClient.post(`${apiPrefix}/follower_indices`, followerIndex).then(extractData)
 );
 
+export const pauseFollowerIndex = (id) => {
+  const ids = arrify(id).map(_id => encodeURIComponent(_id)).join(',');
+  return httpClient.put(`${apiPrefix}/follower_indices/${encodeURIComponent(ids)}/pause`).then(extractData);
```

Comment:
I think it would be best to use a `POST` HTTP request for actions to execute on a resource (and be consistent with the ES API).
Also, as we encode in the `map` each id, I don't think we need to encode once again the ids in the URL.

Reply:
I waffled back and forth between `POST` vs `PUT` for these. I went with `PUT` due to the URLs being idempotent. but in general, having these action endpoints is not pure REST-ful design anyways. it's an interesting discussion that I'd like to raise this during our next sync or team meeting - might affect our API design in the future! I am not opposed to changing these to `POST` as a result of the meeting :slightly_smiling_face: